LuaRules/Gadgets/game_share_mode.lua

```diff
@@ -276,8 +323,43 @@ local function AcceptInvite(player,target)
 			invites[target][player] = nil
 		end
 	else
-		spEcho("[Commshare] Invalid invite: " .. player,target .. "!")
+		DebugEcho("[Commshare] Invalid invite: " .. player,target .. "!")
+	end
+end
+
+local function DisposePlayer(playerID) -- clean up this player.
+	local name,_ = spGetPlayerInfo(playerID,false)
```

Comment:
no need to assign to a useless variable if nothing useful follows
```suggestion
	local name = spGetPlayerInfo(playerID,false)
```

Reply:
Hum. Interesting to note.